### DISCUSSION TITLE
about pbdCloth

### DISCUSSION THREAD

**Discussion by monkeygame** [NVIDIA-Omniverse/PhysX#43] [Q&A] [answered]
When I was testing SnippetPBDCloth_64 and found that after a long run, cloth would be passing through an obstacle, what could I do to improve the simulation accuracy and stability?
![image](https://user-images.githubusercontent.com/15630906/203769416-e9531915-8588-40a5-bc84-9cac09ef781b.png)

**Comment by adenzler-nvidia** [comment, marked as answer]
Hi @monkeygame 

Thanks for reaching out. We were able to reproduce this, however we observe that the cloth does not fall through the sphere, but is pulled off the sphere through a combination of the momentum introduced by the rotating sphere and the small cubes holding the cloth in place on the ground plane. Thus this behaviour is expected. We expect this example to become more stable as we refine the underlying simulation. For now, reducing the rotation speed of the sphere should help avoiding this behaviour.
